### PR TITLE
commented out fseek and fflush in write

### PR DESCRIPTION
**Pull request by masnes**
This passes all but the append test on my machine. Do you want to give it a shot and see if it works on your mac? 

Maybe we don't need to flush for this one.

### CODE REVIEW

**Comment by ianks** [comment]
Everything but appending works. It seems that `do_cyrpt()` is failing for some reason :sob:

**Comment by masnes** [comment]
What I can tell with some manual investigation is that the offset isn't working. Ex:

```
$ make mount
$ echo "testing1" >> mir/newtest.txt
$ cat mir/newtest.txt
testing1
$ echo "testing2" >> mir/newtest.txt
$ cat mir/newtest.txt
testing2
```

But pwrite is helping somewhat, because when I use fseek and fwrite the file is completely borked after any appending. 

I'm not entirely certain the way appending sort of working is good though, because it might be that writing to the offset is what makes the encryption fail. So when the encryption is failing it might be indicating that we're writing to an offset, but not exactly the right one .

File: pa5-encfs.c [hunk 1]
```diff
@@ -376,8 +376,8 @@ static int xmp_write(const char *fuse_path, const char *buf, size_t size,
 	if (res == -1)
```
Comment (ianks) [inline pa5-encfs.c:399]: What does the buf look like after pwrite?

`fprintf(stderr, "Buf: %s\n", buf)`

You should be able to see this in debug mode

**Comment by masnes** [comment]
I've done some additional debugging. What I've found so far:

We had a problem, notably that do_crypt wasn't being called due to a mistake I made with the logic. I've fixed that issue, but now do_crypt is producing an error. I've put in some tracing that makes it easy to see where the error is occuring in the do-crypt function (check out aes-crypt.c). Run with debug and you'll see a trace.

**Comment by ianks** [comment]
Good worked. I'm pooped for tonight but we will get after this with a fresh head tomorrow.

**Comment by ianks** [comment]
:+1:

**Pull request merged**